docs/conf.py

```diff
@@ -2,9 +2,9 @@
 import sys
 import os
 
-sys.path.append('./')
+sys.path.append(os.path.abspath('.'))
```

Comment:
May I ask why do we need this change? This file comes from the Canonical starter pack, so I think if it's absolutely necessary, we should leave it untouched and do things in `custom_conf.py`. Also, I had a look at Pebble docs where it also enabled `sphinx-autobuild`, but there wasn't this change and it worked.

Reply:
docs/conf.py assumes that sphinx is run or loads config with a specific current working directory.

plain sphinx does indeed so that.

somehow sphinx-autobuild does not.

there's another way to fix this, if necessary.